### PR TITLE
Feature that allows us to set menu item more customizable

### PR DESCRIPTION
**Pull request by BranislavMateas**
Hey everyone, me again! Soooo, I found myself in a position where i needed to set custom spacings and height for the menu items. So I added this feature so everyone can do so. Hope you'll like it. Have a great day everyone!

Signed by [Branislav Mateáš](https://github.com/BranislavMateas).

### CODE REVIEW

File: lib/src/side_menu_style.dart [hunk 1]
```diff
@@ -42,6 +42,18 @@ class SideMenuStyle {
...
+  double itemOuterSpacing;
+
+  /// Inner spacing of menu item
+  double itemInnerSpacing;
+
+  /// Height of menu item
+  double itemHeight;
+
+  /// Border Radius of menu item
+  double itemBorRadius;
```
Comment (Jamalianpour) [inline lib/src/side_menu_style.dart]: There is a wrong spelling, fix it

**Comment by BranislavMateas** [comment]
I think it should be fixed now ;)

Signed by [Branislav Mateáš](https://github.com/BranislavMateas).

**Comment by Jamalianpour** [comment]
In `itemBorderRadius` it's better to get `BorderRadius` instead of double.
And I thinks it's better if use Padding for space.
So user can do more customization

**Comment by BranislavMateas** [comment]
Well, with your first point I can agree upon, however with second sentence, I see a problem as your innerPadding isn't really Padding as you're solving this inner spacing by SizedBox, which is not as good imo. I can rewrite it if necessary but for now, I kept the `itemInnerSpacing` - double type, but changed `itemOuterSpacing` to `itemOuterPadding` as it is the only true padding - EdgeInsetGeometry type. Have a great day.

Signed by [Branislav Mateáš](https://github.com/BranislavMateas).

**Pull request merged**